microservices/nris_api/backend/app/nris/scheduled_jobs/nris_jobs.py

```diff
@@ -0,0 +1,48 @@
+from app.extensions import db, sched
+from app.nris.utils.apm import register_apm
+from app.nris.etl.nris_etl import import_nris_xml, clean_nris_xml_import, etl_nris_data, clean_nris_data
+from app.constants import ETL, TIMEOUT_24_HOURS
+from flask import current_app
+from random import randint
+from time import sleep
+import sys
+import cx_Oracle
+
+# the schedule of these jobs is set using server time (UTC)
+
+
+def _schedule_NRIS_ETL_jobs(app):
```

Comment:
I raised this during the implementation in core api, and just raising it again that we can replace this whole method and registering it in init file by using decorators provided by apscheduler
https://github.com/viniciuschiele/flask-apscheduler/blob/master/examples/decorated.py